portugues/colabora/comdiferencas/diff/diff.md

```diff
@@ -1,3 +1,5 @@
 Certo, agora que encontrou as diferencas na versão que introduziu o bug você:
 
+[Corrige as alterações e realiza um "pull request" ao master branch](pull/pull.md)
 
+[Nao sabe corrigir o bug, entao envia ao desenvolvedor as informacoes que conseguiu ate aqui?](../../revisa/revisar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.